tests/macro_read/unknown_attribute.stderr

```diff
@@ -0,0 +1,35 @@
+error: Unknown field: `a`
+ --> $DIR/unknown_attribute.rs:4:8
+  |
+4 | #[deku(a)]
+  |        ^
+
+error: Unknown field: `a`
+ --> $DIR/unknown_attribute.rs:9:12
+  |
+9 |     #[deku(a = 1)]
+  |            ^^^^^
```

Comment:
latest rust stable does
```rust
9 |     #[deku(a = 1)]
  |                  ^
```

Reply:
Yeah, but don't know why it was changed in nightly. I will use stable next time.